plugins/kubernetes/README.md

```diff
@@ -132,11 +132,11 @@ Via [Template filler](/plugins/kubernetes/app/models/kubernetes/template_filler.
 
 ### Migrations / Prerequisite
 
-Add a role with only a `Pod`, `metadata.annotations.samson/prerequisite: 'true'`, and command to run a migrations.
-It will be executed before the rest is deployed.
+Add a role with a `Pod` or `Job`, `metadata.annotations.samson/prerequisite: 'true'`, and `command` to run a migrations.
+It will be executed before any other role is deployed.
```

Comment:
still not clear you need to add this to the job and not the pod template

Reply:
same here...That description is still not clear.